core/runtime.rs

```diff
@@ -184,19 +181,18 @@ pub(crate) struct JsRuntimeState {
   pub(crate) op_ctxs: Box<[OpCtx]>,
   pub(crate) shared_array_buffer_store: Option<SharedArrayBufferStore>,
   pub(crate) compiled_wasm_module_store: Option<CompiledWasmModuleStore>,
-  /// The error that was passed to an explicit `Deno.core.terminate` call.
+  /// The error that was passed to an `op_dispatch_exception` call.
   /// It will be retrieved by `exception_to_err_result` and used as an error
   /// instead of any other exceptions.
-  pub(crate) explicit_terminate_exception: Option<v8::Global<v8::Value>>,
+  // TODO(nayeemrmn): This should be a queue, mirroring `pending_promise_exceptions`.
+  // The REPL doesn't terminate on the first one and should keep track of many.
+  pub(crate) dispatched_exception: Option<v8::Global<v8::Value>>,
```

Comment:
Could you make it a queue in this PR?

Reply:
Done. I hesitated because I figured the polling should be moved to somewhere proper first rather than giving `exception_to_err_result()` a side-effect, but I guess `Option` has the same problem..